src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs

```diff
@@ -482,60 +479,6 @@ private string GetHelpLink(DiagnosticData item)
                         return null;
                     }
 
-                    public bool TryCreateColumnContent(int index, string columnName, bool singleColumnView, out FrameworkElement content)
-                    {
-                        content = default(FrameworkElement);
-                        if (columnName != ShimTableColumnDefinitions.ErrorCode)
-                        {
-                            return false;
-                        }
-
-                        var item = GetItem(index);
-                        if (item == null)
-                        {
-                            return false;
-                        }
-
-                        Uri unused;
-                        if (BrowserHelper.TryGetUri(item.HelpLink, out unused))
-                        {
-                            content = GetOrCreateTextBlock(ref _errorCodes, this.Count, index, item, i => GetHyperLinkTextBlock(i, new Uri(i.HelpLink, UriKind.Absolute), bingLink: false));
```

Comment:
No we still construct link and give it to editor

Sent from my Windows Phone
________________________________
From: Shyam N<mailto:notifications@github.com>
Sent: ý3/ý10/ý2015 10:54 AM
To: dotnet/roslyn<mailto:roslyn@noreply.github.com>
Cc: HeeJae Chang<mailto:Heejae.Chang@microsoft.com>
Subject: Re: [roslyn] don't handle code link ourselves but let error list handle it (#1161)


In src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.cs<https://github.com/dotnet/roslyn/pull/1161#discussion_r26146971>:

> -                        content = default(FrameworkElement);
> -                        if (columnName != ShimTableColumnDefinitions.ErrorCode)
> -                        {
> -                            return false;
> -                        }
> -
> -                        var item = GetItem(index);
> -                        if (item == null)
> -                        {
> -                            return false;
> -                        }
> -
> -                        Uri unused;
> -                        if (BrowserHelper.TryGetUri(item.HelpLink, out unused))
> -                        {
> -                            content = GetOrCreateTextBlock(ref _errorCodes, this.Count, index, item, i => GetHyperLinkTextBlock(i, new Uri(i.HelpLink, UriKind.Absolute), bingLink: false));


Does this mean we no longer construct the bing hyperlink based on 'MessageFormat'? How does error list construct the hyperlink?

We still construct hyperlink based on MessageFormat in preview pane - do we need to change that?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1161/files#r26146971>.